lib/foreman_remote_execution/engine.rb

```diff
@@ -27,7 +29,7 @@ class Engine < ::Rails::Engine
 
     initializer 'foreman_remote_execution.register_plugin', after: :finisher_hook do |_app|
       Foreman::Plugin.register :foreman_remote_execution do
-        requires_foreman '>= 1.10'
+        requires_foreman '>= 1.11'
```

Comment:
Notice the change in line 17 about how migrate paths are loaded, the Railties API changed in a non-backwards compatible way. Other than that, I tried to keep changes backwards-compatible.